packages/parity-electron/src/checkClockSync.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+const sntp = require('sntp');
+
+export const MAX_TIME_DRIFT = 10000; // seconds
+
+export const checkClockSync = () => {
```

Comment:
In TS, this function has type `const checkClockSync: () => any`. Could you add typings so that it returns a `Promise<object{...}>` like in the docs?